roles/deploy_olm_operator/tasks/main.yml

```diff
@@ -233,6 +233,19 @@
     - name: "Failed on deploy_olm_operator_results"
       fail:
         msg: "{{ deploy_olm_operator_results }}"
+    
+    - name: "Get replicaset logs for operators that failed olm-deployment output"
+      shell: "{{ oc_bin_path }} get replicasets -o go-template={%raw%}'{{range .items}}{{.status.conditions}}{{"\n"}}{{end}}'{%endraw%} -n {{ openshift_namespace }}"
+      register: olm_deployment_output
```

Comment:
Renaming this to `olm_replicaset_output` would make it more clear what was being collected.

Reply:
I agree.